packages/client/src/entities/game/controller/utils.ts

```diff
@@ -1,2 +1,2 @@
-export const getRandomNumber = (max: number, min: number) =>
-  Math.floor(Math.random() * max + min);
+export const getRandomNumber = (maxRatio: number, min: number) =>
+  Math.floor(Math.random() * maxRatio + min);
```

Comment:
Всё равно непонятно)) Если это диапазон возможных значений, может так и написать: range ? Ну а по-хорошему надо функцию, которая принимает min и max. И возвращает рандом между ними. Суть не изменится, а проще станет)

Reply:
это не диапазон, а просто одно рандомное число. Такая функция - классика, вот даже ссылка на mdn 
![image](https://user-images.githubusercontent.com/72806681/219724064-4e0400da-da20-4423-93cc-e34ad1d675d6.png)

Reply:
На mdn это как раз максимальное число, до которого может быть рандом. А у нас это диапазон от min до максимального числа

Reply:
Я немного не понимаю суть, а как надо-то?

Reply:
Не, ну можно и так оставить. Это наверное вопрос вкусовщины)

Reply:
Я предлагаю так

```suggestion
getRandom(min, max) {
  return Math.floor(Math.random() * (max - min) + min)
}
```

Получается число в промежутке от min до max

Reply:
Переделала на твой способ, думаю так корректнее. Да и вообще, вдруг кому еще понадобится эта функция, чтоб сразу нормально работала)
Только еще единичку прибавила чтоб от max до min, включая обе величины считалось